core/src/main/java/io/kestra/core/serializers/YamlFlowParser.java

```diff
@@ -97,5 +101,41 @@ private Flow readFile(File file) {
             );
         }
     }
+
+    private Flow readString(String input) {
+        try {
+            return mapper.readValue(input, Flow.class);
+        } catch (JsonProcessingException e) {
+            if (e.getCause() instanceof ConstraintViolationException) {
+                throw (ContainedValueException) e.getCause();
```

Comment:
Why do you throw the cause and not the exception itself ? Skipping the root may loose the exception thrown site.